easybuild/easyconfigs/a/ALPS/ALPS-2.3.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'ALPS'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "http://alps.comp-phys.org/"
+description = "ALPS project"
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++03'}
+
+sources = ['%(namelower)s-%(version)s-src.tar.gz']
+source_urls = ['http://alps.comp-phys.org/static/software/releases/']
+
+boostver = '1.63.0'
+pyver = '2.7.12'
```

Comment:
@dolfim the `%(pyver)s` template only works when `Python` is included as an explicit entry in `dependencies`, so

```python
    ('Python', '2.7.12'),
```